### DISCUSSION TITLE
Completion window doesn't show auto import text anymore

### DISCUSSION THREAD

**Discussion by alex-ds13** [mrcjkb/rustaceanvim#348] [Announcements]
Hey there @mrcjkb. Sorry to bother, this has probably nothing to do with rustaceanvim but  I thought that maybe you could help.

Since a couple of weeks ago I've noticed that when trying to complete some text on a rust file it no longer shows the import text for the completion. Previously if you were trying to write the word `Element` for example and there were multiple import possibilities the completion menu would show them in front of the name. For instance if I typed `Ele` it would show all the available completions and there would be one like this `Element<...>~ (use iced::Element) [LSP]` now it just shows `Element<...>~ [LSP]`.

I don't know if this is related to nvim-cmp, cmp-nvim-lsp, rustaceanvim or rust-analyzer. Was hoping that maybe someone else had faced this before and found a solution.

**Comment by mrcjkb** [comment]
Hey 👋 

If your completion plugin is working otherwise, it's more likely to be a rust-analyzer issue. I would suggest asking there 😃

**Reply by alex-ds13** [reply]
Apparently the issue was with 'lspkind.nvim', they made the label details not show by default. Here I was trying to use previous versions of each plugin to pin-point the issue but I had completely forgot I was using this one. Oops! 😄
